cmd/cspc-operator/app/operations.go

```diff
@@ -256,3 +379,27 @@ func (pc *PoolConfig) ClaimBD(bdName string) error {
 	}
 	return nil
 }
+
+func isPoolSpecBlockDevicesGotReplaced(
+	cspcPoolSpec *apis.PoolSpec, cspi *apis.CStorPoolInstance) bool {
+	cspcBlockDeviceMap := getBlockDeviceMapFromRaidGroups(cspcPoolSpec.RaidGroups)
+	for _, rg := range cspi.Spec.RaidGroups {
+		for _, bd := range rg.BlockDevices {
+			if !cspcBlockDeviceMap[bd.BlockDeviceName] {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+func getBlockDeviceMapFromRaidGroups(
```

Comment:
can we name it func as : `getExistingBlockDeviceMapFromRaidGroups` ?

Reply:
I think func name looks good since for any provided raid group this func will give block devices.